components/camel-mail/src/test/java/org/apache/camel/component/mail/MailExtractAttachmentsTest.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2015 EDMI NZ
+ * All rights reserved.
+ *
+ * This software is the confidential and proprietary information of EDMI. 
+ * ("Confidential Information").  You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with EDMI.
+ */
```

Comment:
Can you update the copyright comment of this file to use the ASF one?  
We are not supposed to commit any code which is not licensed with ASF license.